packages/react/src/components/List/ListItem.tsx

```diff
@@ -98,13 +100,26 @@ const ListItem: React.FC<WithAsProp<ListItemProps> & { index: number }> &
     headerMedia,
     media,
     styles,
-    debug,
-    navigable,
-    selectable,
-    selected,
-    truncateContent,
-    truncateHeader,
-    variables,
+  } = props
+
+  const parentProps = useContextSelectors(ListContext, {
```

Comment:
The typings does not seem to be strict enough to allow only the valid keys and values.
Let's verify it together offline.